cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -3079,7 +3124,8 @@ abstract class TranslatedThrowExpr extends TranslatedNonConstantExpr {
       or
       not exists(this.getDestructors()) and
       kind instanceof ExceptionEdge and
-      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge edge))
+      not kind.(ExceptionEdge).isSEH() and
+      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge edge), true)
```

Comment:
I think `true` should be `false` here (for the same reason as above).

Reply:
Done